FrontEnd/SignUpPage/src/components/signup/signup-form/SignUpFormContent.js

```diff
@@ -0,0 +1,267 @@
+                                type='email'
+                                {...register('email', {
+                                    required: errorMessageTemplates['required'],
+                                    pattern: {
+                                        value: emailPattern,
+                                        message: errorMessageTemplates['email']
+                                    }
+                                })}
+                            />
+                        </div>
+                        <div className={styles["signup-form__error"]}>
+                            {errors.email && errors.email.message}
+                        </div>
+                    </div>
+                    {/* Company field */}
+                    <div className={styles["signup-form__column"]}>
+                        <div className={styles["signup-form__label"]}>
+                            <label className={styles["signup-form__label--required"]}>*</label>
+                            <label className={styles["signup-form__label--text"]}>Назва компанії</label>
+                        </div>
+                        <div className={styles["signup-form__field"]}>
+                            <input
+                                className={styles["signup-form__input"]}
+                                type="text"
+                                placeholder='Назва компанії'
+                                {...register('companyName', {
+                                    required: errorMessageTemplates['required']
+                                })}
+                            />
+                        </div>
+                        <div className={styles["signup-form__error"]}>
+                            {errors.companyName && errors.companyName.message}
+                        </div>
+                    </div>
+                </div>
+                {/* Password field & Confirm Password field */}
+                <div className={styles["signup-form__row"]}>
+                    {/* Password */}
+                    <div className={styles["signup-form__column"]}>
+                        <div className={styles["signup-form__label"]}>
+                            <label className={styles["signup-form__label--required"]}>*</label>
+                            <div className={styles["signup-form__label--password"]}>
+                                <label>Пароль</label>
+                                <label className={styles["signup-form__label--hint"]}>(Повинен містити A-Z, a-z,
+                                    0-9)</label>
+                            </div>
+                        </div>
+                        <div className={styles["signup-form__field"]}>
+                            <input
+                                className={styles["signup-form__input"]}
+                                placeholder='Пароль'
+                                type="password"
```

Comment:
Its better to use one kind of quotes. General recommendation is try to format your code using some code formatter